src/config.ts

```diff
@@ -95,13 +99,16 @@ export class Configuration {
   allowedBranches = ".*";
   initialDevelopment = true;
   maxSubjectLength = 80;
+  releaseBranches = /^release\/.*\d+\.\d+\.\d+.*$/;
```

Comment:
This is the default value for the generic "is this branch a release branch" regex.
I suppose we _could_ incorporate the prefix here, but I'm not sure has any added value here. I'd rather not add too much magic in default values.

Reply:
That was clear to me, but shouldn't we by default incorporate the `version-prefix` in it still? This would be by default `""` unless it is specified by an user. In case it would be specified than equally the release branch pattern would include the string the user has declared.

Reply:
I'm thinking the added magic behavior is not worth it and the currently present wildcard suffices in this case.